### PR TITLE
Main page improvement

### LINKED ISSUE #7: [Main page] Create basic layout
Create the basic layout of the "Home" page without interactivity (no js logic like opening a select, or working search box)

The result of the work will be pretty look home page ready for adding logic in next task
Mockup: https://share.cleanshot.com/7L14w36l

Strikethrough items should be removed. Instead of images create white placeholders with shadow (as shown in the mockup)

### PR DESCRIPTION
**Pull request by Winechicken**
close #7

### CODE REVIEW

File: src/components/homePage/section/RecentForm.vue [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<script setup>
+import Gallery from './Gallery.vue'
+import CardBlock from './CardBlock.vue'
+import ControlButton from './ControlButton.vue'
+
+// let changeImage = document.querySelector('#changeImage')
```
Comment (SavaCool122) [inline src/components/homePage/section/RecentForm.vue:6]: Looks like unnecessary comments here

File: src/components/homePage/section/RecentForm.vue [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+				<label for="forms"><h4>Owner :</h4></label>
+				<select id="forms">
+					<option value="user">Personal</option>
+					<option value="public">Public</option>
+					<option value="all" selected>All</option>
+				</select>
+				<hr />
+				<ControlButton>
+					<img
+						id="changeImage"
```
Comment (SavaCool122) [inline src/components/homePage/section/RecentForm.vue:26]: Looks like unnecessary id here

File: src/components/homePage/section/RecentForm.vue [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+					<img src="../../../assets/icons/folder-com.svg" title="Add file" alt="Add file" />
+				</ControlButton>
+			</div>
+		</div>
+		<Gallery>
+			<CardBlock>
+				<div class="cardText">
+					<p>MyForm1</p>
+					<div>
+						<img src="../../../assets/icons/list.svg" alt="" />
```
Comment (SavaCool122) [inline src/components/homePage/section/RecentForm.vue:45]: Les't do better for our developer experience and implement new rule #9

**Review by SavaCool122 — CHANGES_REQUESTED** [review]

File: src/components/homePage/header/MainPageHeader.vue [hunk 4]
```diff
@@ -0,0 +1,78 @@
+<script setup>
+import { RouterLink } from 'vue-router'
+</script>
+<template>
+	<nav>
+		<div class="contents">
+			<h1>Wood-Forms</h1>
+			<RouterLink to="/about">Overview</RouterLink>
+			<div class="inputarea">
+				<button type="submit"><img src="../../../assets/icons/icons8-search.svg" alt="" /></button>
```
[the remark this answers is not in the document]
Comment (Winechicken) [reply src/components/homePage/header/MainPageHeader.vue:10]: done

File: src/components/homePage/header/MainPageHeader.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+				border-radius: 5px;
+				padding: 5px 50px;
+				height: 100%;
+				width: 100%;
+				font-family: var(--font-main);
+				font-size: var(--size-text);
+				color: var(--color-text);
+				box-shadow: 0 0 5px var(--color-hover);
+			}
+			& button {
```
[the remark this answers is not in the document]
Comment (Winechicken) [reply src/components/homePage/header/MainPageHeader.vue:58]: done

File: src/components/homePage/section/CardBlock.vue [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<template>
+	<div class="block">
+		<div class="image"></div>
+		<slot></slot>
```
[the remark this answers is not in the document]
Comment (Winechicken) [reply src/components/homePage/section/CardBlock.vue:4]: done

File: src/components/homePage/section/CreateForm.vue [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+import Gallery from './Gallery.vue'
+import CardBlock from './CardBlock.vue'
+import ControlButton from './ControlButton.vue'
+</script>
+<template>
+	<main>
+		<div class="heading">
+			<h4>Create a Form</h4>
+			<div class="leftPart">
+				<ControlButton>
```
[the remark this answers is not in the document]
Comment (Winechicken) [reply src/components/homePage/section/CreateForm.vue:11]: Thanks

File: src/components/homePage/section/RecentForm.vue (hunk 1 above)
Comment (Winechicken) [reply src/components/homePage/section/RecentForm.vue:6]: removed

File: src/components/homePage/section/RecentForm.vue (hunk 2 above)
Comment (Winechicken) [reply src/components/homePage/section/RecentForm.vue:26]: this id will be used later when we'll be implementing view changing by button click

Comment (SavaCool122) [reply src/components/homePage/section/RecentForm.vue:21]: please fix

Comment (Winechicken) [reply src/components/homePage/section/RecentForm.vue]: done

**Review by SavaCool122 — APPROVED** [review]

**Pull request merged**